cmd/authctl/user/enable.go

```diff
@@ -0,0 +1,34 @@
+package user
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/internal/consts"
+	"github.com/ubuntu/authd/internal/proto/authd"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// EnableCmd is a command to enable a user.
+var EnableCmd = &cobra.Command{
+	Use:   "enable",
+	Short: "Enable a user managed by authd",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Printf("Enabling user %s\n", args[0])
+		conn, err := grpc.NewClient("unix://"+consts.DefaultSocketPath, grpc.WithTransportCredentials(insecure.NewCredentials()))
+		if err != nil {
+			return err
+		}
+
+		client := authd.NewNSSClient(conn)
```

Comment:
Maybe group all this to a re-usable function

Reply:
Um okay. I will do that. Once we decide if we are going to keep this stuff in NSS or we want to create a separate service.

Reply:
I have done this. Thanks for suggesting @3v1n0.